package/libgluonutil/src/libgluonutil.h

```diff
@@ -27,10 +27,16 @@
 #ifndef _LIBGLUON_LIBGLUON_H_
 #define _LIBGLUON_LIBGLUON_H_
 
+#include <netinet/in.h>
+#include <arpa/inet.h>
```

Comment:
The `arpa/inet.h` in include can be removed here now.

Reply:
done.